CRM/Contact/BAO/Relationship.php

```diff
@@ -1315,7 +1317,7 @@ public static function getRelationship(
     $order = $limit = '';
     if (!$count) {
       if (empty($params['sort'])) {
-        $order = ' ORDER BY civicrm_relationship_type_id, sort_name ';
+        $order = ' ORDER BY wt ';
```

Comment:
Maybe leave the sort_name in, e.g. "ORDER BY wt, sort_name", otherwise for something like `employee of` if they have 30 employees the employees will be listed in random order within the `employee of` subgrouping.